packages/react-components/src/components/MicrophoneButton.tsx

```diff
@@ -171,9 +176,12 @@ export const MicrophoneButton = (props: MicrophoneButtonProps): JSX.Element => {
       menuProps={props.menuProps ?? generateDefaultDeviceMenuPropsTrampoline(props, strings)}
       menuIconProps={props.menuIconProps ?? !enableDeviceSelectionMenuTrampoline(props) ? { hidden: true } : undefined}
       split={props.split ?? enableDeviceSelectionMenuTrampoline(props)}
-      ariaDescription={
+      aria-roledescription={
```

Comment:
should this be ariaRoleDescription? Have we used hyphens in other react props?

Reply:
Not all aria tags have a camel case reflection in react, in this case we have to use the default one.